hal/sparcv8leon3/gaisler/generic/_init.S

```diff
@@ -274,3 +267,10 @@ _init_core_loop:
 	ba _init_core_loop
 	nop
 .size _init_core, . - _init_core
+
+
+.section ".bss"
+.align 4
+_copied_syspage:
+	.skip 0x400
```

Comment:
I think `.zero 0x400` might be a better suited choice

Reply:
Doesn't really matter (GAS manual):
> `.zero`
> This directive is actually an alias for the ‘.skip’ directive

Reply:
I know, it just fits better for .bss. On the other hand, we don't zero-out .bss anyway, so you can leave it as is.

Reply:
Changed to `.zero`